pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -1724,7 +1725,9 @@ func AddSidecarPVCs(log logr.Logger, existing, sidecarPVCs []corev1.PersistentVo
 
 // ExtraPVCVolumes generates Kubernetes volumes from ExtraPVC configurations.
 // Each ExtraPVC references an existing PersistentVolumeClaim by name.
-func ExtraPVCVolumes(log logr.Logger, extraPVCs []ExtraPVC) []corev1.Volume {
+func ExtraPVCVolumes(ctx context.Context, extraPVCs []ExtraPVC) []corev1.Volume {
+	logger := ctrl.LoggerFrom(ctx)
```

Comment:
we are already importing log package, we can just do `log.FromContext(ctx)`

Reply:
oups, correct